Userland/Services/WebContent/WebDriverConnection.cpp

```diff
@@ -19,6 +19,21 @@
 
 namespace WebContent {
 
+#define DRIVER_TRY(expression)                            \
+    ({                                                    \
+        auto _temporary_result = (expression);            \
+        if (_temporary_result.is_error()) [[unlikely]]    \
+            return { _temporary_result.release_error() }; \
+        _temporary_result.release_value();                \
+    })
```

Comment:
This looks identical to regular `TRY`?

Reply:
oh, I see, the return is wrapped in curly braces. How unfortunate :sweat_smile:

Reply:
Feels like a QoL improvement for LibIPC would be that if you're going to send over just one value, you can go ahead and return that value instead of wrapping it